### PR TITLE
[CCXDEV-14260] Fix timeout in CI

### PR DESCRIPTION
**Pull request by juandspy**
# Description

Make sure the notifications-engine doesn't time out.

## Type of change

- Configuration update

## Testing steps

CI

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

**Review by epapbak — APPROVED** [review]

File: pr_check.sh [hunk 1]
```diff
@@ -21,6 +21,7 @@ EXTRA_DEPLOY_ARGS="\
   --set-parameter ccx-notification-writer/IMAGE=quay.io/cloudservices/ccx-notification-writer \
   --set-parameter ccx-notification-db-cleaner/IMAGE=quay.io/cloudservices/ccx-notification-writer \
 "
+EXTRA_DEPLOY_ARGS="${EXTRA_DEPLOY_ARGS} --no-remove-resources notifications-engine"
```
Comment (matysek) [inline pr_check.sh:24]: Pls add a comment about what this option exactly does. It bypasses the usual lower cpu/mem quota for deployment and dedicates more resources for deployment.

Comment (juandspy) [reply pr_check.sh:27]: PTAL: https://github.com/RedHatInsights/ccx-notification-writer/pull/503/commits/66e257d2ca92d5cd94da7401aea13d04200a9d1c

**Review by JiriPapousek — APPROVED** [review]

**Pull request merged**